cmd/podman/common/specgen.go

```diff
@@ -308,9 +308,8 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 	//
 	// Precedence order (higher index wins):
 	//  1) env-host, 2) image data, 3) env-file, 4) env
-	env := map[string]string{
-		"container": "podman",
-	}
+	env := envLib.DefaultEnvVariables()
```

Comment:
It should be getting these fields from containers/common

Reply:
We can change that in a future release; for now, we need this merged so we can get it in 2.0.5

Reply:
Refactored a bit.

I don't think these make sense for containers.conf, we want to ensure a few basic environment variables are always populated - the user shouldn't really need to ever set this, outside of the existing Env section of containers.conf.